gatsby/src/components/headerBody/index.js

```diff
@@ -25,7 +26,9 @@ const HeaderBody = ({ title, subtitle, description, slug, contributors, featured
         <p className="article-subhead">
           {description}
         </p>
-
+        <p className="review-date">
+          <time datetime={lastReviewed} pubdate="pubdate">{lastReviewed}</time>
```

Comment:
`datetime={lastReviewed}` needs to be converted back to ISO YYYY-MM-DD. @alexfornuto please and thank you!

Reply:
Completed in 1febf42576b11887441de87c88967ff883a96894